SC_redesign/Crop_and_Soil/soil/phosphorus_cycling/manure.py

```diff
@@ -0,0 +1,316 @@
+from typing import Optional
+from math import exp, sqrt
+
+from SC_redesign.Crop_and_Soil.soil.soil_data import SoilData
+from SC_redesign.Crop_and_Soil.crop_and_soil_constants import MILLIMETERS_TO_CENTIMETERS, KILOGRAMS_TO_GRAMS, \
+    KILOGRAMS_TO_MILLIGRAMS, HECTARES_TO_SQUARE_CENTIMETERS, HECTARES_TO_SQUARE_MILLIMETERS, CUBIC_MILLIMETERS_TO_LITERS
+
+"""
+This module adds and tracks manure phosphorus dynamics based on the SurPhos model.
+"""
+
+
+class Manure:
+
+    def __init__(self, soil_data: Optional[SoilData] = None):
+        """This method initializes the SoilData object that this module will work with, or create one if none provided.
+
+        Parameters
+        ----------
+        soil_data : SoilData, optional
+            The SoilData object used by this module to track manure phosphorus activity, creates new one if one is not
+            provided.
+
+        """
+        self.data = soil_data or SoilData()
+
+    # --- Static Methods ---
+    @staticmethod
+    def _determine_temperature_factor(avg_air_temperature: float) -> float:
+        """Calculates the temperature factor for the current day
+
+        Parameters
+        ----------
+        avg_air_temperature : float
+            The average air temperature of the current day (degrees celsius).
+
+        Returns
+        -------
+        float
+            The temperature factor on the current day (unitless)
+
+        References
+        ----------
+        SurPhos [2], pseudocode_soil [S.5.D.I.1]
```

Comment:
How do I interpret this SurPhos reference? Do I go to page 2, section 2?

Reply:
No, SurPhos uses square brackets to number the equations. I could make it 
```suggestion
        SurPhos eqn. [2], pseudocode_soil [S.5.D.I.1]
```
if it is currently not clear enough

Reply:
I think the addition of "eqn." would make it a bit clearer, but if SurPhos uses square brackets, the original is fine.

Reply:
Ok, I'm going to leave them as-is then